yarn.lock

```diff
@@ -170,6 +177,13 @@
   version "1.2.2"
   resolved "https://registry.yarnpkg.com/@types/range-parser/-/range-parser-1.2.2.tgz#fa8e1ad1d474688a757140c91de6dace6f4abc8d"
 
+"@types/redis@*":
```

Comment:
run yarn install

Reply:
I do that, but always this row is added into `yarn.lock`.